src/status_im/common/standard_authentication/password_input/view.cljs

```diff
@@ -22,9 +23,11 @@
 
 (defn- on-change-password
   [entered-password]
-  (rf/dispatch [:set-in [:profile/login :password]
-                (security/mask-data entered-password)])
-  (rf/dispatch [:set-in [:profile/login :error] ""]))
+  (debounce/debounce-and-dispatch [:set-in [:profile/login :password]
```

Comment:
unrelated, should we not move the debounce and dispatch to the re-frame utils? I can't see how they're separate 🤷‍♂️